### DISCUSSION TITLE
Should user policies be sorted by when they were first run, or when they were last updated?

### DISCUSSION THREAD

**Discussion by anth-volk** [PolicyEngine/policyengine-app#1698] [Ideas]
Curious to hear thoughts. We currently sort from most recently created to least recently.

**Comment by MaxGhenis** [comment]
most recent at top makes sense to me. is the question whether that should be by create date, update date (what does this mean?), or run date? maybe view date?

**Reply by anth-volk** [reply]
By update (that's the terminology we currently use in the UserProfilePage), I meant run/view, both of which apply to that currently. Should the wording be different?

**Reply by anth-volk** [reply]
And yes, the question is should we order by creation date or view date
